src/site/content/en/fast/incorporate-performance-budgets-into-your-build-tools/index.md

```diff
@@ -107,26 +136,24 @@ By default, **it tests gzipped asset sizes**. You can use the compression option
   </figcaption>
 </figure>
 
-[Lighthouse](https://developers.google.com/web/tools/lighthouse/) is an auditing tool that tests sites in a few key areas — performance, accessibility, best practices and how well your site performs as a progressive web application. You can run it in Chrome DevTools, from the command line, or as a Node module.
+[Lighthouse Bot](https://github.com/ebidel/lighthouse-ci) integrates with Travis CI and enforces budgets based on any of the five Lighthouse audit categories. It’s sometimes simpler to keep an eye on a single number than individual asset budgets and Lighthouse scores take many things into account.
```

Comment:
After the first sentence, I'd add a brief example. (e.g. "For example, you could set a budget of 90 for your Lighthouse performance score.")